packages/venia-ui/lib/components/Gallery/addToCartButton.js

```diff
@@ -24,7 +24,9 @@ const OutOfStockIcon = (
 );
 
 const AddToCartButton = props => {
+    const { urlSuffix } = props;
```

Comment:
Small request.
Could we put `item` in this deconstruction too?

Reply:
Fixed!